plugins/arangodbendorsements/store.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "context"
+
+// Store represents an interface towards any suitable Database store
+type Store interface {
+	// IsInitialised returns an error if store is not initialized
+	IsInitialised() error
+
+	// Connect makes a connection to the store prior to running any queries
+	Connect(ctx context.Context) error
+
+	// GetQueryParam returns the desired query parameter
+	GetQueryParam(input interface{}) (string, error)
```

Comment:
I'm not sure of the reason why `input` is of type `interface{}`.  Better if we can use a stricter typing.

Reply:
Will be resolved using issue #39